pipeline/dbt/models/intermediate/odspep/int_odspep__duplicate_structures.sql

```diff
@@ -0,0 +1,30 @@
+WITH ressources_partenariales AS (
+    SELECT * FROM {{ ref('int_odspep__enhanced_res_partenariales') }}
+),
+
+unique_id_res AS (
+    SELECT DISTINCT ON (id_res)
+        id_res,
+        nom_structure,
+        l1_identification_dest_adr,
+        l3_complement_adr,
+        l4_numero_lib_voie_adr,
+        code_commune_adr
```

Comment:
Tu pourrais mettre la clé qui permet de grouper dans une colonne et de la réutiliser par la suite:
```suggestion
        code_commune_adr,
        UNACCENT(LOWER(CONCAT(nom_structure,
            l1_identification_dest_adr,
            l3_complement_adr,
            l4_numero_lib_voie_adr,
            code_commune_adr))) AS "group_key"
```

Reply:
une autre solution serait d'utiliser une variable dbt